projects/storefrontlib/cms-components/cart/cart-shared/cart-item/model/cart-item-context.model.ts

```diff
@@ -2,7 +2,11 @@ import { Injectable } from '@angular/core';
 import { FormControl } from '@angular/forms';
 import { OrderEntry, PromotionLocation } from '@spartacus/core';
 import { Observable } from 'rxjs';
-import { CartItemComponentOptions } from '../cart-item.component';
+
+export interface CartItemComponentOptions {
+  isSaveForLater?: boolean;
+  optionalBtn?: any;
+}
```

Comment:
are these models (the ones in this file in general) eventually moving out to cart lib?

Reply:
yes, it should go to root.